shopping-cart/shopping-cart-scala/shopping-cart/src/main/resources/application.conf

```diff
@@ -27,3 +27,13 @@ akka.actor.serialization-bindings {
   "akka.actor.Address"        = akka-misc
   "akka.remote.UniqueAddress" = akka-misc
 }
+
+
+akka.actor {
+
+  serialization-bindings {
+    # commands won't use play-json but Akka's jackson support
+    "com.example.shoppingcart.impl.ShoppingCartCommandSerializable"    = jackson-json
```

Comment:
We need this marker-trait for the commands so we can use Akka's Jackson support. 

The main reason being that commands now may have an `ActorRef[Reply]` and we need to have an instance of `ActorSystem[_]` to serialize/deserialize it. 

On a previous experiment, I have tried to build a `PlayJsonSerializer` that could inject the `ActorSystem[_]`, but failed because of cyclic dependencies. We may find other means to do it, but we can eventually leave that out for now.